pam/integration-tests/integration_test.go

```diff
@@ -46,17 +46,20 @@ func TestCLIAuthenticate(t *testing.T) {
 
 	tests := map[string]struct {
 		tape string
+
+		wantLogContents []string
```

Comment:
Forgot to remove the additional field (I do this everytime as well :smiling_face_with_tear: )